q2_quality_control/_pipelines.py

```diff
@@ -8,6 +8,7 @@
 def decontam_identify_batches(ctx, table, metadata,
                               split_column,
                               method,
+                              rep_seqs=None,
```

Comment:
This input doesn't actually seem to be optional based on testing on the command line:

```shell
qiime quality-control decontam-score-viz --i-decontam-scores decon_default_score_table.qza --i-table decon_default_ASV_table.qza --o-visualization viz.qzv --verbose
Traceback (most recent call last):
  File "/Users/jgc/miniconda3/envs/q2dev-amplicon/lib/python3.9/site-packages/q2cli/commands.py", line 520, in __call__
    results = self._execute_action(
  File "/Users/jgc/miniconda3/envs/q2dev-amplicon/lib/python3.9/site-packages/q2cli/commands.py", line 581, in _execute_action
    results = action(**arguments)
  File "<decorator-gen-801>", line 2, in decontam_score_viz
  File "/Users/jgc/miniconda3/envs/q2dev-amplicon/lib/python3.9/site-packages/qiime2/sdk/action.py", line 342, in bound_callable
    outputs = self._callable_executor_(
  File "/Users/jgc/miniconda3/envs/q2dev-amplicon/lib/python3.9/site-packages/qiime2/sdk/action.py", line 615, in _callable_executor_
    ret_val = self._callable(output_dir=temp_dir, **view_args)
  File "/Users/jgc/Documents/4-git-repos/qiime2/q2-quality-control/q2_quality_control/_threshold_graph/_visualizer.py", line 66, in decontam_score_viz
    for seq in rep_seqs:
TypeError: 'NoneType' object is not iterable

Plugin error from quality-control:

  'NoneType' object is not iterable

See above for debug info.
```

Can you address this and add a unit test that covers this case?

Reply:
A unit test in which a 'none' was passed in for the rep_seqs object was written

addressed in [973e000](https://github.com/qiime2/q2-quality-control/pull/82/commits/973e0009e469174b383036bf6faacf0f9c7dd28e)